@Throws(IOException::class, SerializationException::class)
internal fun foo()